### ISSUE TITLE
Update fork with Apache:master

### ISSUE THREAD

**Issue by ssozonoff** [Guavus/incubator-superset#30]
Hi Guys,

Any plans to merge the latest changes from apache:master into your fork ? I am interested in your ES integration work.

Thanks

**Comment by arpit-agarwal** [comment]
@ssozonoff  Yes we are planing to merge to latest stable release this week. https://pypi.org/project/superset/

Any specific fix your are looking at in apache:superset:master that you want in this fork

**Comment by ssozonoff** [comment]
Well typically the getting started using docker instructions dont match up for example :-) Glad you guys are still working on this.
